src/Components/Blueprints/helpers/onPremToHostedBlueprintMapper.tsx

```diff
@@ -150,6 +150,7 @@ export const mapOnPremToHosted = (
     metadata: {
       parent_id: null,
       exported_at: '',
+      is_on_prem: true,
```

Comment:
@ezr-ondrej

Reply:
Ah, missed that, sorry! Ignore me in that case! :)